docs/modules/migrate/pages/dmt-command-reference.adoc

```diff
@@ -0,0 +1,90 @@
+= dmt
+
+dmt commands are a group of migration operations.
+
+[source,bash]
+----
+clc dmt [command] [flags]
+----
+
+== Commands
+
+* <<dmt start, dmt start>>
+* <<dmt status, dmt status>>
+* <<dmt cancel, dmt cancel>>
+* <<dmt estimate, dmt estimate>>
+
+== dmt start
+
+Starts a new migration with the given configuration.
+
+Usage:
+
+[source,bash]
+----
+dmt start [dmt-config-path] [flags]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`--output-dir`, `-o`
+|Optional
+|Output directory for the migration report.
+|Current working directory
+
+|`--yes`
+|Optional
+|Skip confirming the migration operation.
+|`false`
+
+|===
+
+== dmt status
+
+Shows the status of the currently running migration.
+
+Usage:
+
+[source,bash]
+----
+dmt status [flags]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`--output-dir`, `-o`
+|Optional
+|Output directory for the migration report.
+|Current working directory
+
+|===
+
+== dmt cancel
+
+Cancels the currently running migration.
```

Comment:
Cancels the migration.

(you can't cancel one that isn't running, and you can only run one at a time, so it isn't necessary to explain that it is the currently-running migration)

Reply:
https://github.com/hazelcast/hz-docs/pull/890/commits/7e48b6af17c9f43c94a9570a9da520a87ec82cf1